src/components/application_manager/src/commands/mobile/show_constant_tbt_request.cc

```diff
@@ -181,6 +181,7 @@ void ShowConstantTBTRequest::on_event(const event_engine::Event& event) {
   switch (event.id()) {
     case hmi_apis::FunctionID::Navigation_ShowConstantTBT: {
       LOG4CXX_INFO(logger_, "Received Navigation_ShowConstantTBT event");
+      const char* return_info = NULL;
```

Comment:
@malirod , info must be changed only in success case, otherwise NULL, if there will std::string, there will be additional check, so why I can't use const char*?